packages/dev/core/src/Meshes/Builders/capsuleBuilder.ts

```diff
@@ -20,11 +20,11 @@ export function CreateCapsuleVertexData(
         capSubdivisions: 6,
     }
 ): VertexData {
-    const subdivisions = Math.max(options.subdivisions ? options.subdivisions : 2, 1);
-    const tessellation = Math.max(options.tessellation ? options.tessellation : 16, 3);
+    const subdivisions = Math.max(options.subdivisions ? options.subdivisions : 2, 1) | 0;

```

Comment:
did you mean to use bitwise and not logical or?

Reply:
bitwise to ensure integer values